metadata_service/proxy/atlas_proxy.py

```diff
@@ -124,6 +146,27 @@ def _get_table_entity(self, *, table_uri: str) -> Tuple[EntityUniqueAttribute, D
             raise NotFoundException('Table URI( {table_uri} ) does not exist'
                                     .format(table_uri=table_uri))
 
+    def _get_reader_entity(self, table_uri: str, user_id: str) -> EntityUniqueAttribute:
+        """
+        Fetch a Reader entity from parsing table uri and user id.
+        :param table_uri:
+        :param user_id: Qualified Name of a user
+        :return:
+        """
+        table_info = self._extract_info_from_uri(table_uri=table_uri)
+        reader_qn = '{}.{}.metadata.{}.reader@{}'.format(table_info.get('db'),
+                                                         table_info.get('name'),
+                                                         user_id,
+                                                         table_info.get('cluster'))
+
+        try:
+            return self._driver.entity_unique_attribute(
+                "Reader", qualifiedName=reader_qn)
+        except Exception as ex:
+            LOGGER.exception(f'Reader not found. {str(ex)}')
```

Comment:
@morpheyesh I still believe we should create `Reader` entity at least in this place. If the reader entity is not there, create one for this user.